app/models/authorization.rb

```diff
@@ -20,7 +20,7 @@ def default_account
   end
 
   def token_auth_accounts
-    token_ids = token.authorized_resources.try(:keys)
+    token_ids = (token.resources(:member) + token.resources(:admin)).uniq
```

Comment:
Won't this break as soon as PRX/id.prx.org#166 lands?  I thought the plan was to have our apps just pull account_ids out of the token and disregarding scopes (pretty much what we're doing now anyways) ... until we get some real scopes defined in PRX/internal#523 ?

And then sometime after deploying granular-scoped tokens ... we'd come back and lock down our server apps to inspect the actual scope strings attached to each account.  At which point this would become `token.resources(:read_private)`, and probably rename the method to something more specific like `token_auth_readable_accounts`.

Reply:
I think that for the purposes of compatibility the scope list will continue to include the role-name (admin or member) so that we also don't open the flood gates to bad behavior.

and yes, after deployment of id changes, this has to be updated.